src/program/mod.rs

```diff
@@ -31,6 +32,7 @@ pub struct Program<'a> {
     screen: Screen<'a>,
     paths: Paths,
     ui_state: ui::State,
+    sorter: Sorter,
```

Comment:
I thought the goal was to have it ready so that it would be usable in execution?
Though, I guess it's easy to just add it back in or somewhere else when using it at run time.

Reply:
Fixed